code/game/gamemodes/cult/cult_mode.dm

```diff
@@ -172,11 +172,15 @@ GLOBAL_LIST_EMPTY(all_cults)
 		to_chat(cult_mind.current, "<span class='motd'>For more information, check the wiki page: ([GLOB.configuration.url.wiki_url]/index.php/Cultist)</span>")
 		return TRUE
 
-/datum/game_mode/proc/remove_cultist(datum/mind/cult_mind, show_message = TRUE, remove_gear = FALSE)
+/datum/game_mode/proc/remove_cultist(datum/mind/cult_mind, show_message = TRUE, remove_gear = FALSE, mob/target_mob)
 	if(!(cult_mind in cult)) // Not actually a cultist in the first place
 		return
 
-	var/mob/cultist = cult_mind.current
+	var/mob/cultist
+	if(!target_mob)
+		cultist = cult_mind.current
+	else
+		cultist = target_mob
```

Comment:
Could this be simplified to...?
```suggestion
	var/mob/cultist = target_mob
	if(!cultist)
		cultist = cult_mind.current
```

Reply:
I don't think that's all that readable tbh

Reply:
seems marginally better to me

Reply:
K